docs/paper/admin/reference/global-configuration.md

```diff
@@ -86,6 +75,28 @@ function itself. For per-world configuration, see the
 - **default**: `100.0`
 - **description**: The maximum number of chunks sent to an individual player within one second.
 
+## chunk-system
+
+### gen-parallelism
+
+- **default**: `default`
```

Comment:
Does this default to default? Seems odd - maybe just put true here

Reply:
Yea, the default value is `default` :smile: